packages/connect-react/src/hooks/form-context.tsx

```diff
@@ -395,6 +436,98 @@ export const FormContextProvider = <T extends ConfigurableProps>({
     checkPropsNeedConfiguring()
   };
 
+  const handleSdkErrors = (o: unknown[] | unknown | undefined) => {
+    let newErrors = [
+      ...sdkErrors,
+    ]
+    if (!o) return
+    const handleObservationErrors = (observation: never) => {
```

Comment:
It doesn't really handle the error, right? It just extracts some details.
```suggestion
    const getErrorFromObservation = (observation: never) => {
```

Also, why is the type of the observation `never`?